### DISCUSSION TITLE
Backlog sorting & stale tickets

### DISCUSSION THREAD

**Discussion by sflynn850** [raceintospace/raceintospace#854] [General]
Hey all,

Been away for a bit, but I have some free time for work reasons. Thought I'd maybe see if I could go through some old tickets, propose some for closure, and then jump back on getting this to compile in MacOS.

Anyway, does anyone have thoughts on the process for closing old/stale tickets? Right now I can just comment and see if we get a consensus — or, like in the case of tickets like #128, I could just close them and someone can re-open if they disagree.

**Comment by peyre** [comment]
I _love_ the idea of closing old tickets, especially our earliest ones.  I think you're right on #128 and just closed it.

Please do.  It'd be good to do some cleanup.  And working on the Mac side would be great, as we have people working on preparing a Windows and a Linux release.

**Reply by sflynn850** [reply]
Great! I'll run through the backlog again and see what we can whittle down. Maybe start tagging things with a 2.1 or 2.2 release, so we can get our hands around it a little more.

I'm a software architect in my day job, and while I don't have a ton of C++ experience, I do have "sort and organize" experience :)

**Reply by peyre** [reply]
Good!  Do you think you'd be in a position to prepare a Mac release in a bit?  The guy who used to do them when we were on SourceForce no longer answers emails.

**Reply by sflynn850** [reply]
Certainly, if I can get it to build on MacOS. Issues #781 and #347 are currently blocking a build on MacOS, so I'd need to resolve those before a release. I'm thinking a MacOS version might be a good target for 2.1, rather than blocking 2.0.

**Reply by peyre** [reply]
That's great news, thanks.  It'd be nice if we could have all three in the 2.0 release.

**Reply by sflynn850** [reply]
Agreed - hopefully I can sort it out relatively quickly.
